public/js/main.js

```diff
@@ -0,0 +1,32 @@
+$(document).ready(function () {
```

Comment:
I have added the comments for this file above. Remove the extra file which is not being used

Reply:
Done.